server/webhook.go

```diff
@@ -337,6 +336,13 @@ func (s *Server) processFile(ctx context.Context, pushEvent *vcs.PushEvent, repo
 			)
 			return "", false, nil, nil
 		}
+		if migrationInfo == nil {
+			log.Error("File path does not match file path template",
```

Comment:
include path.Join(repo.BaseDirectory, repo.FilePathTemplate) in the log

Reply:
Done.